### ISSUE TITLE
Applying rule on specific portion of the page

### ISSUE THREAD

**Issue by amitguptagwl** [avanslaars/cypress-axe#15]
`cy.checkA11y( ".example-class" )` runs the rule on whole page. I want to do a11y testing only one specific portion of the page. Do I need to pass any other configuration?

**Comment by BriarP** [comment]
I've encountered this as well when attempting to run checka11y on just one area of the page.

**Comment by amitguptagwl** [comment]
@avanslaars any update on this?

**Comment by pgroot91** [comment]
@amitguptagwl tried reproducing your reported issue but i'm unable to reproduce the behavior. For me its working as expected! 

I have eg an violation on a specific part in the app in this case the Facebook button.

`cy.checkA11y(".facebook-btn")`

Log:
```
[
  {
    "any": [
      {
        "id": "color-contrast",
        "data": {
          "fgColor": "#ffffff",
          "bgColor": "#1778f2",
          "contrastRatio": 4.19,
          "fontSize": "12.0pt (16px)",
          "fontWeight": "normal",
          "expectedContrastRatio": "4.5:1"
        },
        "relatedNodes": [
          {
            "html": "<li class=\"facebook-btn\">",
            "target": [
              ".facebook-btn"
            ]
          }
        ],
        "impact": "serious",
        "message": "Element has insufficient color contrast of 4.19 (foreground color: #ffffff, background color: #1778f2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
      }
    ],
    "all": [],
    "none": [],
    "impact": "serious",
    "html": "<span>Sign in with Facebook</span>",
    "target": [
      "a[title=\"Sign\\ in\\ with\\ Facebook\"] > span:nth-child(2)"
    ],
    "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 4.19 (foreground color: #ffffff, background color: #1778f2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
  }
]
```

**Comment by amitguptagwl** [comment]
I'm using the latest version. Which version are you using?

**Comment by pgroot91** [comment]
I need to double check my package file tomorrow but guess latest version of cypress and cypress-axe.

**Comment by pgroot91** [comment]
The versions installed on my environment:
```
"axe-core": "^3.3.1",
"cypress": "^3.4.1",
"cypress-axe": "^0.5.1"
```

**Comment by amitguptagwl** [comment]
I was using the selector in the wrong way. My bad
